frontend/apply/routes.py

```diff
@@ -0,0 +1,10 @@
+from flask import render_template
+
+from pre_award.common.blueprints import Blueprint
+
+apply_bp = Blueprint("apply_routes", __name__, template_folder="templates")
+
+
+@apply_bp.route("/monolith")
+def landing_page():
+    return render_template("apply/landing-mono.html.jinja")
```

Comment:
Do we actually want this to be a routable thing on the live deployed service?

Reply:
Probs not!